scimma_admin/scimma_admin/urls.py

```diff
@@ -16,12 +16,13 @@
 from django.contrib import admin
 from django.urls import include, path
 from django.http import HttpResponse
+from hopskotch_auth import views
 
 def OK(request):
     return HttpResponse("OK")
 
 urlpatterns = [
-    path('', OK),
+    path('', views.index),
```

Comment:
Alternatively, we could redirect to `hopauth/`. Not sure there's any practical argument either way.

Reply:
They seem to do basically the same thing as far as I can tell, and this way skipped having to write the (admittedly very simple) view which just did a redirect.